daemon/daemon.go

```diff
@@ -907,9 +907,21 @@ func (daemon *Daemon) Shutdown() error {
 // (is it not set coming in? why is it unset?)
 func (daemon *Daemon) Mount(container *container.Container) error {
 	dir, err := container.RWLayer.Mount(container.GetMountLabel())
```

Comment:
mounted_layer which implements the Mount() call knows nothing about containers. Its only aware of mounts. Only daemon knows about containerIDs, which are necessary to create the "mount-path" file under the container directory.